modules/oneVideoBidAdapter.md

```diff
@@ -40,6 +40,7 @@ Connects to Verizon Media's Video SSP (AKA ONE Video / Adap.tv) demand source to
                   inventoryid: 123,
                   minduration: 10,
                   maxduration: 30,
+                  ttl: 300
```

Comment:
seems like you are missing comma `,` after `ttl: 300`

Reply:
thanks for catching :) fixed accordingly